content/integrations/uwsgi.md

```diff
@@ -0,0 +1,76 @@
+---
+categories:
+  - log collection
+  - web 
+doc_link: https://docs.datadoghq.com/integrations/uwsgi/
+description: Collect uWSGI logs in order tracks requests per second, bytes served, request status and more.
+short_description: Collect uWSGI logs in order tracks requests per second, bytes served, request status and more.
+ddtype: check
+git_integration_title: uwsgi
+has_logo: true
+integration_title: uwsgi
+is_public: true
+kind: integration
+name: uwsgi
+public_title: Integration Datadog-uWSGI
+short_description: 
+supported_os:
+- linux
+- mac_os
+- windows
+---
+
+## Overview
+
+Collect uWSGI logs in order tracks requests per second, bytes served, request status (2xx, 3xx, 4xx, 5xx), service uptime, slowness, and more.
```

Comment:
Collect uWSGI logs in order, track requests per second, bytes served, request status (2xx, 3xx, 4xx, 5xx), service uptime, slowness, and more.

also change the `description` and `short_description` above to have grammar consistency.